### DISCUSSION TITLE
Close eventStream server side

### DISCUSSION THREAD

**Discussion by jaschaio** [sergiodxa/remix-utils#193] [Q&A] [answered]
Hey there,
I am using the `eventStream` and `useEventSource` as explained in the documentation but wonder how I can close the request server side, e.G. stop streaming?

Client Side I came up with the Following but it is kind of ugly and throws a 404 Not Found Error:

```js
const [ url, setUrl ] = useState( '/sse/time' );

let live = useEventSource( url, { event: 'time' } );

useEffect( () => {

    setTimeout( () => setUrl(), 5000 );

}, [] );
```

**Comment by sergiodxa** [comment, marked as answer]
I wrote about how to do this here https://sergiodxa.com/tutorials/close-sse-connection-from-the-server-in-remix

**Reply by jaschaio** [reply]
Thanks! I got hung up on trying to use the same signal instead of just using my own AbortController.

Unrelated: I have been trying to reach you about a remix hiring opportunity but can't find a good email for you. What's a good way to reach you via private message?

Thanks again

**Reply by sergiodxa** [reply]
I may have missed it, I'm going to search it with your name and answer you.
